### ISSUE TITLE
preview-tui bat not using specified theme

### ISSUE THREAD

**Issue by arihans** [jarun/nnn#1160]
#### Environment details

- [x] Operating System: Pop!OS 21.04 x86_64
- [x] Desktop Environment: Gnome 3.38.4
- [x] Terminal Emulator: Kitty 0.19.3
- [x] Shell: zsh 5.8
- [x] Configuration options set:
```
export BAT_THEME="Dracula"
export NNN_PLUG='f:finder;d:diffs;p:preview-tui;t:treeview'
export NNN_FIFO=/tmp/nnn.fifo
export PAGER='less -RF'
```
- [x] Issue exists on `nnn` master

#### Exact steps to reproduce the issue

1. Install the latest release of `nnn`
2. Install the `nnn preview-tui` plugin
3. Install `bat`
4. Create a `bat -> batcat` symlink ( since `bat` is named batcat in Ubuntu distros )
5. export BAT_THEME="<Any theme from `bat --list-theme` other than default>"
6. No changes to the `bat` theme in the preview whatsoever
---
Adding `--theme=Dracula` (My preferred theme) to 
```
ln  176 : bat --terminal-width="$(tput cols)" --decorations=always --color=always \
                    --paging=never --style="$BAT_STYLE" "$@" &
```
does solve the issue. but `--theme=$BAT_THEME` does nothing. echoing `$BAT_THEME` inside `preview-tui` scirpt prints nothing showing that it is inaccessible inside `preview-tui`.
`bat` is working fine with the specified theme outside of nnn.

**Comment by jarun** [comment]
@luukvbaal can you please if this is resolved by https://github.com/jarun/nnn/pull/1141?

**Comment by luukvbaal** [comment]
It would be yeah. Although on my system it inherits `BAT_THEME` just fine from my profile, not sure why you would need to set it dynamically.

**Comment by jarun** [comment]
@arihans can you check why you can't inherit the theme dynamically?

**Comment by arihans** [comment]
@jarun it seems kitty splits can't access some of the environment variables. I've got no idea why. 
#### Method of testing - 
```
$ kitty @ launch --type=tab ./printenv_less
```
It can be solved easily by adding `--env "BAT_THEME=$BAT_THEME"` while lauching kitty split [here](https://github.com/jarun/nnn/blob/fa7cef2df75c940994f4a700b317645034864b26/plugins/preview-tui#L122).
It works perfectly with tmux.

**Comment by Kabouik** [comment]
I suppose it is related to what is explained [here](https://github.com/jarun/nnn/pull/1004#issuecomment-841201353) and discussed in messages below?

**Comment by luukvbaal** [comment]
Yeah to be honest I forgot about the environment difference between `kitty` and `tmux`, just merge #1411 imo.

**Comment by jarun** [comment]
Closed at #1141.
